scene/main/scene_tree.cpp

```diff
@@ -1106,11 +1106,11 @@ int SceneTree::get_node_count() const {
 
 void SceneTree::_update_root_rect() {
 	if (stretch_mode == STRETCH_MODE_DISABLED) {
-		_update_font_oversampling(1.0);
-		root->set_size((last_screen_size / stretch_shrink).floor());
+		_update_font_oversampling(stretch_scale);
+		root->set_size((last_screen_size).floor());
 		root->set_attach_to_screen_rect(Rect2(Point2(), last_screen_size));
-		root->set_size_override_stretch(false);
-		root->set_size_override(false, Size2());
+		root->set_size_override_stretch(true);
+		root->set_size_override(true, (last_screen_size / (stretch_scale)).floor());
```

Comment:
Isn't it a bit weird that `STRETCH_MODE_DISABLED` sets the `size_override_stretch` to `true`?

Reply:
Well, without enabling stretch it would be impossible to scale things up. However, the only stretching that happens is the user-defined scaling, so I would argue that there is no automatic scaling.
Perhaps `STRETCH_MODE_MANUAL` would be a better name than `STRETCH_MODE_DISABLED` but that would probably be a rather confusing and major change.